lessons/advanced/otp-concurrency.md

```diff
@@ -149,79 +147,120 @@ iex> SimpleQueue.queue
 [1, 2, 3, 20]
 ```
 
-For more information check out the official [GenServer](http://elixir-lang.org/docs/stable/elixir/GenServer.html#content) documentation.
+### A Simple Cron Server
```

Comment:
What are you thinking would be good to go beyond it? Thats where I started but I didn't really get any ideas I liked.